packages/markdown/index.js

```diff
@@ -81,7 +107,7 @@ export function normalize(blocks) {
     .replace(/\[block:/g, '\n\n[block:')
     .replace(/\[\/block\]/g, '[/block]\n')
     .trim()
-    .replace(/^(#+)([\w\s\d]+\n\n)/gm, '$1 $2');
+    .replace(/^(#+)((?:\w|\d)[^{#\n]+\n{1,})/gm, '$1 $2');
```

Comment:
😬

Reply:
_Oh_ yeah, I know... but I honestly don't know how else to address these edge-case situations? I've been trying to keep all my shittiest decisions in this normalization function so at least we can trace 'em down.

Reply:
Haha, I didn't mean that as a slam. If it works it works!